test/extended/cloud_controller_manager/ccm.go

```diff
@@ -88,6 +92,38 @@ var _ = g.Describe("[sig-cloud-provider][Feature:OpenShiftCloudControllerManager
 				o.HaveField("Contents", o.ContainSubstring("cloud-provider=external")),
 			)))
 	})
+
+	g.It("Cluster scoped load balancer healthcheck port and path should be 10256/healthz", func() {
```

Comment:
The PR looks good, just want to check if that port could ever change, if so you'll save yourself some headaches in having to go submit a test rename PR if you take the port name itself out of the test name and keep it more generic.

Reply:
The port number is the "kube-proxy" health check port. We don't actually use kube-proxy, but instead OVN-K exposes a health check endpoint at the same port.

If we were to change the port, this would mean all CCMs would have to be updated with a new port, and all implementations/distros of Kube would also need to update

In short, I don't see the port number changing, but, we could change the name anyway just in case